### PR TITLE
[DDW-742]: Implement new delegation countdown design

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR implemented new delegation countdown design
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-742)

## Todos
- [x] Implement new delegation countdown design

## Screenshots
image
image

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1563527934439900)
- [x] Visit `Staking` => `Staking Countdown` to open new countdown screen and check if all the components are correctly rendered.

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by gnpf** [comment]
@yakovkaravelov The font does not match with the sample design. Also, the texts in Japanese are missing.

image

**Comment by yakovkaravelov** [comment]
> @yakovkaravelov The font does not match with the sample design. Also, the texts in Japanese are missing.
> 
> image

You tested it while this had `WIP` label. :)
Now everything is completed.
Please review again.
Thanks.

**Comment by a-rukin** [comment]
image
border-radius is missing (should be 2px)

**Comment by a-rukin** [comment]
image
boxes should be the same size

File: source/renderer/app/components/widgets/CountdownWidget.js [hunk 1]
```diff
@@ -51,39 +91,88 @@ export default class CountdownWidget extends Component<Props, State> {
...
   render() {
-    const timeLeftString = this.translateTimeLeft();
+    const { timeLeft } = this.state;
+    const fieldPanels = this.generateCountdownPanels();
 
-    return <div className={styles.timeLeft}>{timeLeftString}</div>;
+    return (
+      <div className={styles.timeLeftContainer}>
+        <div className={styles.timeLeft}>
+          {timeLeft === 0 && (
```
Comment (tomothespian) [inline source/renderer/app/components/widgets/CountdownWidget.js]: @yakovkaravelov please use if <-> else like `timeLeft === 0 ? (<SVGInline svg={spinnerIcon} className={styles.spinnerIcon} />) : (fieldPanels)`

**Comment by daniloprates** [comment]
@yakovkaravelov please, fix the conflicts and merge the latest `develop`

**Comment by daniloprates** [comment]
Merged via [`vNext`](https://github.com/input-output-hk/daedalus/pull/1504) PR.

**Pull request closed without merging**